doc/integrator/database.rst

```diff
@@ -110,3 +111,11 @@ Publish the new version on production: now, integration and production both use
 For OpenShift projects, just push the integration branch into the production branch.
 
 The schema ``main_2019`` still exists, so if needed, the production can be rolled back to this content.
+
+
+Editing
+~~~~~~~
+
+To have a differant schema for the geodata used in the editing we can defined the geo_table like
```

Comment:
To have a different schema for the geodata used in the editing, we can define the geo_table as follows:

Reply:
Thanks, fixed :-)